crates/astria-bridge-withdrawer/src/metrics.rs

```diff
@@ -97,10 +97,10 @@ impl metrics::Metrics for Metrics {
             )?
             .register()?;
 
-        let batch_total_settled_value = builder
+        let accumulated_settled_value = builder
             .new_gauge_factory(
-                BATCH_TOTAL_SETTLED_VALUE,
-                "Total value of withdrawals settled in a given sequencer block",
+                ACCUMULATED_SETTLED_VALUE,
+                "Total value of withdrawals settled over time from batches",
```

Comment:
Would be good to include the units used here.  Also maybe worth replacing `_VALUE` in the metric name with the units too (or appending it to the name)